fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalPlan.java

```diff
@@ -70,19 +71,25 @@ default FunctionalDependencies computeFuncDeps() {
         computeFd(fdBuilder);
         ImmutableSet<FdItem> fdItems = computeFdItems();
         fdBuilder.addFdItems(fdItems);
+
+        List<Set<Slot>> uniqueSlots = fdBuilder.getAllUnique();
+        Set<Slot> uniformSlots = fdBuilder.getAllUniform();
         for (Slot slot : getOutput()) {
             Set<Slot> o = ImmutableSet.of(slot);
             // all slot dependents unique slot
-            for (Set<Slot> uniqueSlot : fdBuilder.getAllUnique()) {
+            for (Set<Slot> uniqueSlot : uniqueSlots) {
                 fdBuilder.addDeps(uniqueSlot, o);
             }
             // uniform slot dependents all unique slot
```

Comment:
this comment should be `uniform slot dependents all slot`?

Reply:
Yeah, you are right. I'll fix it in the next pr